contracts.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+package patrol;
+
+option java_package = "pl.leancode.automatorserver.contracts";
+
+message OpenAppCommand { string appId = 1; }
+
+message TapCommand {
+  string appId = 1;
+  Selector selector = 2;
+}
+
+message DoubleTapCommand {
+  string appId = 1;
+  Selector selector = 2;
+}
+
+message EnterTextBySelectorCommand {
+  string appId = 1;
+  string data = 2;
+  Selector selector = 3;
+}
+
+message EnterTextByIndexCommand {
+  string appId = 1;
+  string data = 2;
+  uint32 index = 3;
+}
+
+message TapOnNotificationByIndexCommand { uint32 index = 1; }
+
+message TapOnNotificationBySelectorCommand { Selector selector = 1; }
+
+message SwipeCommand {
+  float startX = 1;
+  float startY = 2;
+  float endX = 3;
+  float endY = 4;
+  uint32 steps = 5;
+}
+
+message HandlePermissionCommand {
+  Code code = 1;
+
+  enum Code {
+    WHILE_USING = 0;
+    ONLY_THIS_TIME = 1;
+    DENIED = 2;
+  }
+}
```

Comment:
all the "commands" and "queries" are gonna get GRPC-ed in the next PR (or maybe this one? idk).